torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1865,8 +1872,19 @@ def get_types_for_tangents(tangents):
                         )
                         for i, t in enumerate(all_args)
                     ]
+
+                    metas = iter(CompiledFunction.metadata.subclass_tangent_meta)
+                    tangent_metadata = [
```

Comment:
this is unused? (we also care about runtime perf - and this will involve an extra loop over a list of size [num_tangents])

Reply:
Yes, it is unused. I'll remove it